docs/sources/mimir/operators-guide/monitor-grafana-mimir/dashboards/ruler/index.md

```diff
@@ -17,3 +17,9 @@ The Ruler dashboard shows health and activity metrics for the ruler and object s
 The following example shows a Ruler dashboard from a demo cluster.
 
 ![Grafana Mimir ruler dashboard](mimir-ruler.png)
+
+Note: Even while operating in [Remote ruler mode]({{< relref "../../../architecture/components/ruler/#remote" >}}) you will still see values for the `Read from ingesters - QPS` and `Write to ingesters - QPS` panels.
+
+This is because the metrics are inclusive of intermediate services and are showing the values that ultimately reach the ingesters.
+
+For a more detailed view of the read path when using remote ruler mode, see the [Remote ruler reads]({{< relref "../remote-ruler-reads-resources/" >}}) dashboard.
```

Comment:
WDYT about making this change in the description field of the dashboard itself?

Reply:
@dimitarvdimitrov How about both? I know I came to this doc page looking for the answer and didn't find it, but I could see people also trying to find the answer within the dashboard.

Reply:
@osg-grafana is OOO; @dimitarvdimitrov if you approve, please merge.

Reply:
sounds good